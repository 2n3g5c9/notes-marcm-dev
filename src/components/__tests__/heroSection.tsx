import React from 'react'
import renderer from 'react-test-renderer'

import { HeroSection } from '@components'

describe('HeroSection', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<HeroSection />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
