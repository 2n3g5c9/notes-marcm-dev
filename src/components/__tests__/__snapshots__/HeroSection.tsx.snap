// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroSection renders correctly 1`] = `
<div
  className="bg-hero bg-secondary max-w-full px-4 md:px-14 py-10 sm:py-12 md:py-14 lg:py-20 xlish:py-9"
>
  <div
    className="max-w-6xl mx-auto pb-5 relative overflow-hidden"
  >
    <div
      className="flex content-center max-w-6xl relative z-10 lg:max-w-3xl lg:w-full"
    >
      <div
        className="flex-grow m-auto px-3 lg:px-8"
      >
        <div
          className="sm:text-center lg:text-left"
        >
          <h1
            className="font-bold font-serif text-primary text-4xl sm:text-5xl lg:text-6xl"
          >
            <span
              className="inline"
            >
              Title
              Â 
            </span>
          </h1>
          <p
            className="mt-3 text-base text-secondary sm:mt-5 sm:text-lg sm:max-w-xl sm:mx-auto md:mt-5 lg:mx-0"
          >
            Subtitle
          </p>
          <div
            className="text-base font-medium mt-5 sm:mt-8 sm:flex sm:justify-center lg:justify-start"
          >
            <div
              className="rounded-md shadow"
            >
              <a
                className="w-full flex items-center justify-center border border-transparent rounded-md text-tertiary bg-accent hover:bg-hover-accent px-8 py-3 lg:py-3.5 lg:text-lg"
                href="#"
              >
                Read the notes
              </a>
            </div>
            <div
              className="mt-3 sm:mt-0 sm:ml-3"
            >
              <a
                className="w-full flex items-center justify-center border border-transparent rounded-md text-dark-accent bg-light-accent hover:bg-hover-light px-8 py-3 lg:py-3.5 lg:text-lg"
                href="/about"
              >
                More about me
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        className="animate-wiggle flex-none w-6 hidden xlish:block"
      />
    </div>
  </div>
</div>
`;
